server/src/main/resources/config/windows/nodeConfig.json

```diff
@@ -29,6 +29,7 @@
         "port": 5555,
         "host": "ip",
         "register": true,
-        "hubPort": 4444
+        "hubPort": 4444,
+        "nodeRecycleThreadWaitTimeout": 300
```

Comment:
Building on my previous note.. Configuration for SeLion servlets such as `TransferServlet` or process handlers such as `WindowsProcessHandler` wouldn't make sense to put in the selenium hub configuration -- these things don't read from / have access to the same registry, for one.. Thus, `SeLionConfig.json` and/or command-line dash options processed by SeLion-Grid is more suitable for their configuration.